packaging/dpkg_selections

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python
+
+DOCUMENTATION = '''
+---
+module: dpkg_selections
+short_description: Dpkg package selection states
+description:
+    - Change dpkg package state via --get-selections and --set-selections.
+version_added: "1.6"
+author: Brian Brazil <brian.brazil@boxever.com>
+options:
+    name:
+        description:
+            - Name of the package
+        required: true
+    state:
```

Comment:
I would rename to selection to avoid confusing with the normal meaning of 'state' in ansible modules